web/Web/web/core/session_utils.py

```diff
@@ -44,14 +47,50 @@ def submit_new_session_form(request):
             topic=form.instance,
             max_number_of_judgments=max_number_of_judgments,
             strategy=strategy,
-            show_full_document_content=show_full_document_content
+            show_full_document_content=show_full_document_content,
         )
         messages.add_message(request,
                              messages.SUCCESS,
                              success_message)
         request.user.current_session = session
         request.user.save()
 
+        try:
+            if 'topic-judgments_file' in request.FILES:
+                judgments_file = TextIOWrapper(request.FILES['topic-judgments_file'].file, encoding='utf-8')
```

Comment:
Let's have the code to read the csv file similar to what I have in https://github.com/UWaterlooIR/gathera/blob/26a3c0bf622d52710034ba1a88cc95127c7f479b/web/Web/web/judgment/views.py#L385

It gives warnings to the user if the file is corrupted or in the wrong format.